index.html

```diff
@@ -95,12 +95,12 @@ <h4>Wins</h4>
                     <div class="card_main_program">
                         <i class="fa-solid fa-user-plus"></i>
                         <h4>Deliver</h4>
-                        <p>With all the map of challenges, opportunities and process we activate execution!</p>
+                        <p>With the map of challenges, opportunities and process we activate execution!</p>
```

Comment:
- [ ] kindly ensure that the desktop about page does not have the partners sections. i has a dark background with a logo and text. please fix that
- [ ] please kindly increase the size(height) of your images so the match the original design. great job


![Screenshot (623)_LI](https://user-images.githubusercontent.com/65298048/203655052-344b3be5-42eb-4682-9e2b-eac5bb9b574c.jpg)